hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -528,7 +533,10 @@ public String toString() {
     }
   }
 
-  private static String toSortedTruncatedString(TypedProperties props) {
+  static String toSortedTruncatedString(TypedProperties props) {
+    List<String> sensitiveConfigList = props.getStringList(HoodieWriteConfig.SENSITIVE_CONFIG_KEYS_FILTER.key(),
```

Comment:
Why can't it be a Set? Not that it matters for few handful of elements, but Set feels more intuitive.

Reply:
We are iterating through all the elements so kept a list. I can change to a LinkedHashSet if we want to avoid duplicates?

Reply:
A simple HashSet would do but it doesn't matter much in this case. Generally speaking, whenever I need to do contains check, I prefer set.